Tiltfile

```diff
@@ -721,14 +728,52 @@ if wormchain:
         ref = "wormchaind-image",
         context = ".",
         dockerfile = "./wormchain/Dockerfile",
+        build_args = {"num_guardians": str(num_guardians)},
         only = [],
         ignore = ["./wormchain/testing", "./wormchain/ts-sdk", "./wormchain/design", "./wormchain/vue", "./wormchain/build/wormchaind"],
     )
 
-    k8s_yaml_with_ns("wormchain/validators/kubernetes/wormchain-guardian-devnet.yaml")
+    def build_wormchain_yaml(yaml_path, num_instances):
+        wormchain_yaml = read_yaml_stream(yaml_path)
+
+        # set the number of replicas in the StatefulSet to be num_guardians
+        wormchain_set = set_replicas_in_statefulset(wormchain_yaml, "wormchain", num_instances)
+
+        # add a Service for each wormchain instance
+        services = []
+        for obj in wormchain_set:
+            if obj["kind"] == "Service" and obj["metadata"]["name"] == "wormchain-0":
+
+                # make a Service for each replica so we can resolve it by name from other pods.
+                # copy wormchain-0's Service then set the name and selector for the instance.
+                for instance_num in list(range(1, num_instances)):
+                    instance_name = 'wormchain-%s' % (instance_num)
+
+                    # Copy the Service's properties to a new dict, by value, three levels deep.
+                    # tl;dr - if the value is a dict, use a comprehension to copy it immutably.
+                    service = { k: ({ k2: ({ k3:v3
```

Comment:
maybe use deepcopy?
```
import copy
# ...
service = copy.deepcopy(obj)
```

Reply:
I tried _real_ hard to import copy, I could not get it working with tilt `v0.30.5`. Does importing like that work for you?

I looked into starlark docs and it seems like it should just be there, you don't have to import standard libs? They are just in the global namespace it seems (not sure though). I'm not sure if Tilt is not exposing them, or what. I'm open to whatever thoughts here.